Docker/Dockerfile.navstack.ubuntu-galactic

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+FROM ros:galactic-ros-base-focal
+
+ENV WORKSPACE=/data/workspace
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+ENV O3DE_DEMO_REPO=https://github.com/RobotecAI/o3de-demo-project.git
+
+WORKDIR $WORKSPACE
+
+RUN apt-get update && apt-get upgrade -y
+
+# Add additional package repositories needed for packages
+RUN apt-get install -y --no-install-recommends gpg wget
+
+#
+# Install packages needed for cloning and building from the source repos
+#
+
+# InLaunchNavStackstall the git and build tools
+RUN apt-get install -y --no-install-recommends git 
+
+# Add additional ROS2/Galactic packages
+RUN /bin/bash -c 'source /opt/ros/galactic/setup.bash && \
+                  apt install --no-install-recommends -y ros-${ROS_DISTRO}-desktop \
+                                                         ros-${ROS_DISTRO}-slam-toolbox \
+                                                         ros-${ROS_DISTRO}-navigation2 \
+                                                         ros-${ROS_DISTRO}-nav2-bringup \
+                                                         ros-${ROS_DISTRO}-pointcloud-to-laserscan \
+                                                         ros-${ROS_DISTRO}-gazebo-msgs \
+                                                         ros-${ROS_DISTRO}-ackermann-msgs \
+                                                         ros-${ROS_DISTRO}-control-toolbox'
+
+# Install the O3DE-specific required packages
+RUN apt-get install -y --no-install-recommends libglu1-mesa-dev \
+                                               mesa-common-dev \
+                                               libnvidia-gl-470
```

Comment:
Isn't libnvidia-gl-470 kind of old?
It looks like it goes up to 520 now https://packages.ubuntu.com/search?suite=jammy&section=all&arch=any&keywords=libnvidia-gl&searchon=names

Reply:
There seems to be a problem with libnvidia-gl-520 (vulkan is not detected  in the container). 470 is known to work